mls.json

```diff
@@ -950,7 +952,7 @@
             }
           },
           {
-            "repos": ["w3c/clreq", "w3c/jlreq", "w3c/klreq"]
+            "repos": ["w3c/clreq", "w3c/jlreq", "w3c/jlreq-d/", "w3c/klreq"]
```

Comment:
We don't seem to use the public-i18n-cjk list anymore. Should we update this, @r12a?

Reply:
Also, there's an extra slash at the end, which might be the cause of the CI error.